<launch>

    <!-- drone0 -->
    <group ns="drone0">
        <!-- Mavros -->
        <include file="$(find mavros)/launch/px4.launch">
            <arg name="fcu_url" value="$(arg fcu_url)" />
            <arg name="gcs_url" value="$(arg gcs)" />
            <arg name="tgt_system" value="$(eval 1 + arg('ID'))" />
            <arg name="tgt_component" value="1" />
        </include>

        <node pkg="px4_offb" type="downwash" name="downwash" output="screen">
            <param name="uav_id" value="$(arg xid)" />

            <!-- position in ENU, check in VICON -->
            <rosparam param="take_off_position">[0.0, 0.0, 2.37]</rosparam>
            <rosparam param="waypoint_1">[1.0, 1.0, 2.37]</rosparam>
            <rosparam param="land_position">[2.0, 2.0, 2.37]</rosparam>
        </node>

    </group>


</launch>